curriculum/challenges/english/25-front-end-development/quiz-javascript-loops/66edcd49e73385dd4df54ac7.md

```diff
@@ -205,7 +209,7 @@ A loop that continues to run indefinitely because the termination condition is n
 
 #### --text--
 
-For iterating over an array, which loop is not the best?
+If you want to print each element in an array, which loop would be the most inconvinient?
```

Comment:
I am not super in love with this question. I feel like "best" or "most inconvenient" is subjective in people's eyes. 
So I vote a different question.

Reply:
How about something like this:

```
Which of these loops iterates over the values of an array?

--distractors--

`for` loop
`while` loop
`for...in` loop

--answer--

`for...of` loop
```

Reply:
Here's an alternative:

```
How many times will the following loop run?

for (let i = 2; i < 10; i+=2) {
  console.log(i);
}

--distractors--

5
9
10

--answer--

4